src/frontend/src/components/Post/Post.jsx

```diff
@@ -84,6 +93,18 @@ const Post = ({ id, html, author, url, title, date, link }) => {
   return (
     <Container className={classes.root}>
       <ListSubheader className={classes.header}>
+        <div className={classes.iconDiv}>
```

Comment:
This extra UI should only show for an admin user, right?  We need that logic, too.